nodes/modifier_make/bisect.py

```diff
@@ -32,12 +32,12 @@
 # but using python bmesh code for driving
 # by Linus Yng / edits+upgrades Dealga McArdle and Victor Doval
 
-def bisect_bmesh(bm, pp, pno, outer, inner, fill):
+def bisect_bmesh(bm, pp, pno, outer, inner, fill, threshold):
```

Comment:
I would suggest to add a default value for the new parameter here: `threshold = 0.00001` (the same value which was earlier hardcoded). Just in case someone is doing `from sverchok.nodes.modifier_name.bisect import bisect_bmesh` somewhere in scripted node. We do not have any obligations to do not break scripted nodes, but still, if this costs nothing, it's better to preserve compatibility.

Reply:
Append